packages/cinema/data/words/movie.json

```diff
@@ -0,0 +1 @@
iw","country":"Hebrew","string":"סרט"},{"locale":"ja","country":"Japanese","string":"映画"},{"locale":"jw","country":"Javanese","string":"film"},{"locale":"ka","country":"Georgian","string":"ფილმი"},{"locale":"kk","country":"Kazakh","string":"фильм"},{"locale":"km","country":"Khmer","string":"ខ្សែភាពយន្ត"},{"locale":"kn","country":"Kannada","string":"ಚಲನಚಿತ್ರ"},{"locale":"ko","country":"Korean","string":"영화"},{"locale":"la","country":"Latin","string":"movie"},{"locale":"lo","country":"Lao","string":"ຮູບ​ເງົາ"},{"locale":"lt","country":"Lithuanian","string":"filmas"},{"locale":"lv","country":"Latvian","string":"filmu"},{"locale":"mg","country":"Malagasy","string":"sarimihetsika"},{"locale":"mi","country":"Maori","string":"kiriata"},{"locale":"mk","country":"Macedonian","string":"филм"},{"locale":"ml","country":"Malayalam","string":"സിനിമ"},{"locale":"mn","country":"Mongolian","string":"кино"},{"locale":"mr","country":"Marathi","string":"चित्रपट"},{"locale":"ms","country":"Malay","string":"filem"},{"locale":"mt","country":"Maltese","string":"film"},{"locale":"my","country":"Myanmar (Burmese)","string":"ရုပ်ရှင်"},{"locale":"ne","country":"Nepali","string":"चलचित्र"},{"locale":"nl","country":"Dutch","string":"film"},{"locale":"no","country":"Norwegian","string":"film"},{"locale":"ny","country":"Chichewa","string":"kanema"},{"locale":"pa","country":"Punjabi","string":"ਫਿਲਮ"},{"locale":"pl","country":"Polish","string":"film"},{"locale":"pt","country":"Portuguese","string":"filme"},{"locale":"ro","country":"Romanian","string":"film"},{"locale":"ru","country":"Russian","string":"кино"},{"locale":"si","country":"Sinhala","string":"චිත්රපටය"},{"locale":"sk","country":"Slovak","string":"film"},{"locale":"sl","country":"Slovenian","string":"film"},{"locale":"so","country":"Somali","string":"filim"},{"locale":"sq","country":"Albanian","string":"film"},{"locale":"sr","country":"Serbian","string":"филм"},{"locale":"st","country":"Sesotho","string":"filimi"},{"locale":"su","country":"Sundanese","string":"Pilem"},{"locale":"sv","country":"Swedish","string":"film"},{"locale":"sw","country":"Swahili","string":"movie"},{"locale":"ta","country":"Tamil","string":"திரைப்பட"},{"locale":"te","country":"Telugu","string":"సినిమా"},{"locale":"tg","country":"Tajik","string":"филм"},{"locale":"th","country":"Thai","string":"หนัง"},{"locale":"tl","country":"Filipino","string":"pelikula"},{"locale":"tr","country":"Turkish","string":"film"},{"locale":"uk","country":"Ukrainian","string":"фільм"},{"locale":"ur","country":"Urdu","string":"فلم"},{"locale":"uz","country":"Uzbek","string":"film"},{"locale":"vi","country":"Vietnamese","string":"bộ phim"},{"locale":"yi","country":"Yiddish","string":"פֿילם"},{"locale":"yo","country":"Yoruba","string":"fiimu"},{"locale":"zh","country":"Chinese","string":"电影"},{"locale":"zh-CN","country":"Chinese (Simplified)","string":"电影"},{"locale":"zh-TW","country":"Chinese (Traditional)","string":"電影"},{"locale":"zu","country":"Zulu","string":"movie"}]
```

Comment:
Same thing as others `json` files, we should run `prettier` and I don't it's great, to have the data in one single line, not really readable.